packages/landingpage-new/components/explore/resources/resources-card.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { ReactNode } from 'react';
+import styles from './resources-card.module.scss';
+import Image from 'next/image';
+import Link from 'next/link';
+import linkIcon from './assets/link-icon.svg';
+
+type Resource = {
+  id: number;
+  blog_url: string;
+  blog_img_url: string;
+  blog_title: string | ReactNode;
+  blog_author: string;
+};
+
+function ResourceCard() {
+  const cards: Resource[] = [
+    {
+      id: 1,
+      blog_url: 'https://www.inovex.de/de/blog/inovex-elements-open-source/',
```

Comment:
We should match the url to our currently selected locale. 
e.g.: If I set DE as my language I expect the link to point to the german version of the blog entry. The same with english.